pynamodb/attributes.pyi

```diff
@@ -119,6 +119,11 @@ class NumberAttribute(Attribute[float]):
     @overload
     def __get__(self, instance: Any, owner: Any) -> float: ...
 
+class VersionAttribute(Attribute[float]):
+    @overload
+    def __get__(self: _A, instance: None, owner: Any) -> _A: ...
+    @overload
+    def __get__(self, instance: Any, owner: Any) -> float: ...
```

Comment:
Did you consider coercing and typing this as an `int` instead?

Reply:
I didn't since this is really meant to be a _hidden_ attribute, so I just left the same interface that exists for `NumberAttribute`. I'm fine with making these changes if you think it's worth doing though.

Reply:
Although the version is managed for you here, I think it'll still be common to read the value for logging or other purposes, so I do think typing as an int would be helpful